### ISSUE TITLE
`koch xz` tries to pack removed files

### ISSUE THREAD

**Issue by kaushalmodi** [nim-lang/Nim#9451]
`koch xz` seems to try to pack files which no longer exist.

Specifically these drew my attention:

```
[Warning] No file found that matches: web/website.ini
[Warning] No file found that matches: web/ticker.html
[Warning] No file found that matches: web/*.nim
[Warning] No file found that matches: web/*.rst
[Warning] No file found that matches: web/*.csv
[Warning] No file found that matches: web/news/*.rst
```

### Current Output

When running `koch xz`, I see these:

```
Hint: /home/travis/build/kaushalmodi/nightlies/nim/863d3ef4ed7d18877a42f0060ebe63c34a27cf32/tools/niminst/niminst --var:version=0.19.1 --var:mingw=none --main:compiler/nim.nim scripts compiler/installer.ini [Exec]
[Warning] No file found that matches: makefile
[Warning] No file found that matches: install_nimble.nims
[Warning] No file found that matches: install_tools.nims
[Warning] No file found that matches: web/website.ini
[Warning] No file found that matches: web/ticker.html
[Warning] No file found that matches: web/*.nim
[Warning] No file found that matches: web/*.rst
[Warning] No file found that matches: web/*.csv
[Warning] No file found that matches: web/news/*.rst
[Warning] No file found that matches: bin/nimblepkg/*.nim
[Warning] No file found that matches: bin/nimblepkg/*.cfg
[Warning] No file found that matches: dist/nimsuggest
[Warning] No file found that matches: bin/nim.exe
[Warning] No file found that matches: bin/c2nim.exe
[Warning] No file found that matches: bin/nimgrep.exe
[Warning] No file found that matches: bin/nimsuggest.exe
[Warning] No file found that matches: bin/nimble.exe
[Warning] No file found that matches: bin/vccexe.exe
[Warning] No file found that matches: bin/nimgrab.exe
[Warning] No file found that matches: koch.exe
[Warning] No file found that matches: finish.exe
[Warning] No file found that matches: tools\start.bat
[Warning] No file found that matches: bin/makelink.exe
[Warning] No file found that matches: bin/7zG.exe
[Warning] No file found that matches: bin/*.dll
```

### Expected Output

None of those warnings should have appeared.

### Additional Information

https://api.travis-ci.org/v3/job/443967650/log.txt

@Araq

**Comment by nc-x** [comment]
Copying my comment from https://github.com/nim-lang/Nim/pull/9558
  
These warnings still appear on Linux -
```
[Warning] No file found that matches: bin/nim.exe
[Warning] No file found that matches: bin/c2nim.exe
[Warning] No file found that matches: bin/nimgrep.exe
[Warning] No file found that matches: bin/nimsuggest.exe
[Warning] No file found that matches: bin/nimble.exe
[Warning] No file found that matches: bin/vccexe.exe
[Warning] No file found that matches: bin/nimgrab.exe
[Warning] No file found that matches: koch.exe
[Warning] No file found that matches: finish.exe
[Warning] No file found that matches: tools\start.bat
[Warning] No file found that matches: bin/makelink.exe
[Warning] No file found that matches: bin/7zG.exe
[Warning] No file found that matches: bin/*.dll
```

These are windows specific files that koch is trying to look for on Linux. (Similarly on Windows, you will still find warnings for Linux specific files).

**Comment by narimiran** [comment]
Fixed by #9558
